SCClassLibrary/JITLib/ProxySpace/NodeMap.sc

```diff
@@ -173,12 +174,58 @@ ProxyNodeMap : NodeMap {
 	}
 }
 
+
+// represents a subset of the output bus of a proxy
+// that can be mapped to a control.
+
+NodeProxyPartMap {
+	var <proxy, <indices;
+
+	*new { |proxy, indices=0|
```

Comment:
mmmh i am not sure, maybe it can be one index or an array of indices? do you think it is too confusing to allow for both options?

Reply:
I don't think it's confusing to allow both, indices and index, as it is quite common in sclang. But the question is what a default would be. Currently it seems to be the first channel, but maybe all channels are a better default (via `null` which gets replaced with a full array)? But first channel could also be a good default.

Reply:
I think it would be better without a default. The case of the first channel is just as special as any other case for this class.

Indeed, all channels are a better default.